src/components/common/Drawer.jsx

```diff
@@ -0,0 +1,45 @@
+import { Drawer, Box, Typography, IconButton, Icon, Menu } from "@mui/material";
+import MenuIcon from "@mui/icons-material/Menu";
+import ArrowBackIosNewSharpIcon from "@mui/icons-material/ArrowBackIosNewSharp";
+import { styled } from "@mui/material/styles";
+
+import { useState } from "react";
+
+const DrawerHeader = styled("div")(() => ({
+  display: "flex",
+  alignItems: "center",
+  justifyContent: "flex-end",
+}));
+
+export const MuiDrawer = () => {
+  const [isDrawerOpen, setisDrawerOpen] = useState(false);
+  return (
```

Comment:
add and empty line before return, make sure that all components respect this rule.

Reply:
also update into setIsDrawerOpen